src/libs/vehicle/ardupilot/ardupilot.ts

```diff
@@ -388,6 +388,44 @@ export abstract class ArduPilotVehicle<Modes> extends Vehicle.AbstractVehicle<Mo
     return true
   }
 
+  /**
+   * Helper function for changing the altitude of the vehicle
+   * @param {number} altitude (in meters)
+   */
+  _changeAltitude(altitude: number): void {
+    const gotoMessage: Message.SetPositionTargetLocalNed = {
+      time_boot_ms: 0,
+      type: MAVLinkType.SET_POSITION_TARGET_LOCAL_NED,
+      target_system: 1,
+      target_component: 1,
+      coordinate_frame: { type: MavFrame.MAV_FRAME_LOCAL_OFFSET_NED },
+      type_mask: { bits: 0b0000111111111000 },
+      x: 0,
+      y: 0,
+      z: altitude,
+      vx: 0,
+      vy: 0,
+      vz: 0,
+      afx: 0,
+      afy: 0,
+      afz: 0,
+      yaw: 0,
+      yaw_rate: 0,
+    }
+
+    this.write(gotoMessage)
+  }
+
+  /**
+   * Change altitude
+   * @param {number} altitude_septoint
+   * @returns {void}
+   */
+  changeAltitude(altitude_septoint: number): void {
```

Comment:
Typo here. And prefer using camelCase, as it's the default on Js.

Reply:
Nice catch, I'll fix when I get home

Reply:
fixed